### PR TITLE
fix(RELEASE-1064): fix linting issues in create-advisory task

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: tasks/create-advisory/tests/test-create-advisory-fail-no-snapshot.yaml
Comment (mmalina) [inline tasks/create-advisory/tests/test-create-advisory-fail-no-snapshot.yaml:1]: Version bump is needed. Plus a README entry.

File: tasks/create-advisory/create-advisory.yaml [hunk 1]
```diff
@@ -49,20 +49,20 @@ spec:
     - name: run-script
       image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
       script: |
-        #!/bin/sh
+        #!/bin/bash
```
Comment (johnbieren) [inline tasks/create-advisory/create-advisory.yaml:52]: Can you change mocks.sh to use bash instead of sh too?

Comment (jinqi7) [reply tasks/create-advisory/create-advisory.yaml:52]: Updated. Thanks

**Review by johnbieren — DISMISSED** [review]
I think we were just bumping the z so this would be version 4.4.1, not 4.5.0, but I am fine with either way. Feel free to resolve this if you don't want to change it

File: tasks/create-advisory/tests/test-create-advisory-fail-no-snapshot.yaml
Comment (jinqi7) [reply tasks/create-advisory/tests/test-create-advisory-fail-no-snapshot.yaml:1]: Added. Thanks

**Review by johnbieren — APPROVED** [review]

**Pull request merged**